app/src/main/java/com/main/mealplanner/dto/MealPlan.kt

```diff
@@ -8,6 +8,6 @@ import java.util.*
 @Entity(tableName="mealplan")
 data class MealPlan(var CookSchedule: Date, var RecipeId : String, var OwnerEmail : String, @PrimaryKey @SerializedName("id") var MealPlanId:Int = 0) {
     override fun toString(): String {
-        return "Cook Recipe #" + RecipeId + " at " + CookSchedule.toString()
+        return "Cook Recipe # $RecipeId at $CookSchedule"
```

Comment:
A faster way to do this in Kotlin. Same result, less typing.
Source:
https://kotlinlang.org/docs/classes.html